### DISCUSSION TITLE
Sega Master System and the "Pause" Button

### DISCUSSION THREAD

**Discussion by vicsidious** [darthcloud/BlueRetro#915] [Q&A] [answered]
Hi, me again! 
I've been playing some Master System games for a while and recently got to play a game that ABSOLUTELY needs the "Pause" button to enter inventory. 
I always believed it to be impossible to activate the pause button from the controller because it is on the console, but as I fiddled with the blueretro esp32 component, I flashed the Parallel 1P instead of the Parallel 2P required by the SMS. This gave weird inputs and I noticed that I entered the pause function, by the controller, by mistake!! 
I could not achieve the same result with the Parallel 2P firmware, no matter the mapping I chose.

My question is the following: Is there ANY WAY to input a "pause" function to the sega master system, through BlueRetro, without modding the console? Can Base1, Base2 or Base3 functions help in achieving this? 

Thank you, I hope somebody can figure this out :)

**Comment by vicsidious** [comment, marked as answer]
Ok well, turns out I was ill informed. The pause button was not activated at all under the Parallel 1P weird button combination. Turns out it was the second controller D-Pad input that had the same behaviour as the pause button.
i.e: in Wonder Boy 3, pressing the pause button shows the the inventory menu.... unbeknownst to me, the player 2 left and up buttons does the same!! 

Which begs another question..... 

Can we program combos (like multiple inputs) on the same button press?

**Reply by darthcloud** [reply]
Yes you can using the mapping config just use the same source twice with a different destination.
